spec/authority/controller_spec.rb

```diff
@@ -344,6 +344,13 @@ def self.before_filter(*args) ; end
           controller_instance.send(:authorize_action_for, resource_class, options)
         end
 
+        it "passes options set by authority_actions" do
+          options = {:for => 'insolence'}
```

Comment:
:laughing: I forgot I wrote that. I always enjoy putting silly things in tests, doc strings, etc.